scalameta/semanticdb-scalac/src/main/scala/scala/meta/internal/semanticdb/AttributesOps.scala

```diff
@@ -165,10 +165,11 @@ trait AttributesOps { self: DatabaseOps =>
 
                 names(mtree.pos) = symbol
                 if (mtree.isBinder) binders += mtree.pos
-                denotations(symbol) = gsym.toDenotation
+                denotations(symbol) = gsym.toDenotation(symbol)
```

Comment:
I don't see a reason to compute `Symbol.toSemantic` multiple times. This is not because of performance reasons, it pollutes the console when println debugging and the same method is invoked twice with the same arguments.